integreat_cms/cms/forms/regions/region_form.py

```diff
@@ -729,7 +744,7 @@ def duplicate_language_tree(
             )
 
 
-# pylint: disable=too-many-locals
+# pylint: disable=too-many-locals,too-many-arguments
```

Comment:
I think it makes sense to create a dataclass that bundes some of the arguments, so we don't need to disable this line anymore :)

Reply:
But does that not *increase* cognitive overhead?

I get that sometimes, a lot of arguments can indicate that a function simply does too much, but that is not the case here - the parameters are just required.

Since this is (afaik) the only place we need this combination of parameters, extracting them into a dataclass would just mean you additionally need to look up what the dataclass contains, instead of just having the info presented to you where you expect it, in the function signature.

Reply:
Yes, I think you're right :)